src/components/Dashboard/FilingHistoryList/filings/ContinuationOut.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <FilingTemplate class="continuation-out" :filing="filing" :index="index">
+    <template #body>
+      <div v-if="isFilingComplete">
+        <h4>Continuation Out Complete</h4>
+        <p class="mt-4">
+          The Company {{ legalName }} was successfully
+          <strong>
+            Continued Out on {{ continuationOutDate }}, to
+            {{ foreignJurisdiction }} under the name {{ legalName }}.
+          </strong>
+          The Company has been struck from the register and ceased to be an
+          incorporated company under the Business Corporations Act.
+          You are required to retain a copy of all the Continuation Out documents in your records books.
+        </p>
+        <p v-if="fileNumber" class="mt-4 mb-0">Court Order Number: {{ fileNumber }}</p>
+        <p v-if="hasEffectOfOrder" class="mt-0">Pursuant to a Plan of Arrangement</p>
+      </div>
+    </template>
+  </FilingTemplate>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { ApiFilingIF } from '@/interfaces'
+import FilingTemplate from '../FilingTemplate.vue'
+import { DateUtilities, EnumUtilities } from '@/services'
+import { CountriesProvincesMixin } from '@/mixins'
+
+@Component({
+  components: { FilingTemplate }
+})
+export default class ContinuationOut extends Mixins(CountriesProvincesMixin) {
+  @Prop({ required: true }) readonly filing!: ApiFilingIF
+  @Prop({ required: true }) readonly index!: number
+
+  get legalName (): string {
+    return this.filing.data?.continuationOut?.legalName
+  }
+
+  get continuationOutDate (): string {
+    const continuationOutDate = this.filing.data?.continuationOut?.continuationOutDate
+    if (continuationOutDate) {
+      return DateUtilities.apiToPacificDate(continuationOutDate, true)
```

Comment:
I am changing this in API to provide just date (similar to expiry in restoration https://github.com/bcgov/business-filings-ui/blob/main/src/components/Dashboard/FilingHistoryList/filings/LimitedRestoration.vue#L45) since `continuationOutDate` is provided by the user

Reply:
Awesome, that helps a lot actually. Thanks Vysakh! So when I create a filing with a `continuationOutDate` "2023-06-06" for example, will it be stored as is or will it change to API format?